pkg/controller/workload/workload_processor.go

```diff
@@ -671,6 +686,29 @@ func (p *Processor) handleAuthorizationTypeResponse(rsp *service_discovery_v3.De
 	return nil
 }
 
+func (p *Processor) storeWorkloadPolicies(uid string, polices []string) {
+	var (
+		key   = bpf.WorkloadPolicy_key{}
+		value = bpf.WorkloadPolicy_value{}
+	)
+	if len(polices) == 0 {
+		return
+	}
+	key.WorklodId = p.hashName.StrToNum(uid)
```

Comment:
This may break hash we now. We assume all the str now is `workload` or `service` ID, and on restart we depend on it.

Reply:
how we used this hash file to restore?

Reply:
we need a function like `compareWorkloadAndServiceWithHashName` after restart. To handle authz policy deleted during restart.

And there we assume all the strs are workload or services. But now, there is auth policy now, so maybe not influencing the results, just added unneccessary loops


```
	for str, num := range p.hashName.strToNum {
		if p.WorkloadCache.GetWorkloadByUid(str) == nil && p.ServiceCache.GetService(str) == nil {
			log.Debugf("GetWorkloadByUid and GetService nil:%v", str)
```

Reply:
got it

Reply:
fix